src/app/project-overview/template.html

```diff
@@ -118,6 +118,15 @@
     <div fxLayout="column"
          fxLayoutGap="20px"
          fxFlex="50">
+      <mat-card *ngIf="project">
+        <mat-card-header>
+          <mat-card-title>Project Quota</mat-card-title>
```

Comment:
Is this visible on community edition as well?

Reply:
No it isn't, I'm adding checks for enterprise edition.